src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerWaitContainer.groovy

```diff
@@ -15,19 +15,37 @@
  */
 package com.bmuschko.gradle.docker.tasks.container
 
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.Optional
+
+import java.util.concurrent.TimeUnit
+
 class DockerWaitContainer extends DockerExistingContainer {
 
     protected int exitCode
 
-    DockerWaitContainer(){
-        ext.getExitCode = {exitCode}
+    /**
+     * Wait timeout in seconds.
+     */
+    @Input
+    @Optional
+    Long timeout
```

Comment:
Lets change the timeout to an Integer and default to using seconds. I previously said lets use milliseconds but we are using seconds elsewhere so lets do the same here if only for consistencies sake.